app/src/main/AndroidManifest.xml

```diff
@@ -17,110 +17,87 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.movemeet.noActionBar">
+        <activity android:name=".view.activity.SplashActivity"></activity>
```

Comment:
Thank you for having reordered our Manifest! We didn't do it since a long time!

Reply:
Ahaha it's Jean-Luc's job